utils/src/main/kotlin/CopyrightStatementsProcessor.kt

```diff
@@ -112,7 +112,7 @@ class CopyrightStatementsProcessor {
     }
 
     data class Result(
-        val processedStatements: LinkedHashMap<String, List<String>>,
+        val processedStatements: Map<String, List<String>>,
```

Comment:
When you say "no code depends on the specific type" I assume that you have checked that no user of `processedStatements` assumes the key order to be maintained on insertion / removal of keys, correct?

Reply:
Actually, the test case depends on a predictable order, I have switched to `SortedMap` instead, because alphabetic sorting provides some value, while the order of insertion is irrelevant.